### PR TITLE
Companies section reordering and content updates

### LINKED ISSUE #888: Companies Page section reordering, content updates
Mark asked in Slack that dev do the following to the Companies page. 

for context, this page has two sections of content, "Host a Tech-for-Good Hackathon" and "Sponsor Our Platform & Public Events" hereafter just called Hackathon and Sponsor.

 - In both the Hackathon and Sponsor tab, move the carousel titled "What our partners are saying" above the contact form.
 - In the Hackathon tab, rename "[Read about previous events in our blog](https://github.com/DemocracyLab/CivicTechExchange/blob/master/common/components/controllers/CorporateHackathonController.jsx#L270)" to "Learn more in our blog" and change the hyperlink to https://blog.democracylab.org/how-amazon-hacked-for-good/
 - Leave the blog link in the Sponsors section unchanged.
 - Change the link in `_renderHackathonProspectus()` to the latest version: `2022+DemocracyLab+Corporate+Hackathon+Prospectus.pdf`
 - Change the link in `_renderSponsorProspectus()` to the latest version: `2022+DemocracyLab+Sponsorship+Prospectus.pdf`
 - update section background colors (described below)
 
The page design alternates background color per section. This is manually controlled so we'll have to fix it manually as well: Remove or add the `.corporate-bg-light` class to divs as appropriate, and then the carousel will require some additional work. While the overall section will be grey background, the area inside the border should be white background.
 
 Below the 992px breakpoint, it should look like this:

Above the 992px breakpoint, it should look like this:

Feel free to ask me about this if you get stuck, but I'd start with rules on `.corporate-root .carousel-inner` and `.corporate-root .carousel-item` using media-queries to switch behavior at the correct breakpoint -- however you do it, make sure the changes do not affect the carousel on other pages such as the homepage.

relevant files:
https://github.com/DemocracyLab/CivicTechExchange/blob/master/common/components/controllers/CorporateHackathonController.jsx
https://github.com/DemocracyLab/CivicTechExchange/blob/master/civictechprojects/static/css/partials/_CorporateHackathon.scss

### PR DESCRIPTION
**Pull request by schaferyan**
Made the following changes to Companies Page, requested in issue #888 
- Moved Testimonials Carousel above Contact Form in both Hackathon and Sponsorship tabs
- Updated blog link in Hackathon tab
- Updated Prospectus links
- Updated Section background colors

closes #888

### CODE REVIEW

File: common/components/controllers/CorporateHackathonController.jsx [hunk 1]
```diff
@@ -253,24 +253,27 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
           </h3>
         </div>
 
-        {this._renderContactHackathon()}
+        
 
-        <div className="corporate-hackathon-saying corporate-bg-light col-12">
+        <div className="corporate-hackathon-saying col-12 corporate-bg-light">
```
Comment (PeterBreen) [inline common/components/controllers/CorporateHackathonController.jsx:258]: Did you mean to remove corporate-bg-light here or just reorder the class declarations?

File: common/components/controllers/CorporateHackathonController.jsx [hunk 2]
```diff
@@ -253,24 +253,27 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
...
+          <div className="carousel-testimonial-root corporate-bg-light">
             <TestimonialCarousel
               category="partner-highlights"
               interval={600000}
             />
           </div>
           <p className="h3">
-            Read about previous events in our <a href={window.BLOG_URL}>blog</a>
-            .
+            {/* TODO find where {window.BLOG_URL} is declared */}
```
Comment (PeterBreen) [inline common/components/controllers/CorporateHackathonController.jsx:270]: Let's remove this and I'll write an issue that we need to document environment variables better in our Contributor's Guide. I reviewed it this morning and agree that we need a section about how to use them, not just how to load them.

**Review by PeterBreen — CHANGES_REQUESTED** [review]
I took a look and this looks good with one exception - but the section changes, content updates, and blog modifications all look excellent. 

However the section background for the carousel section still shows as white; to properly alternate it should be grey. If you remove the `corporate-bg-light` class from the controller (see the line comments I made) that ought to expose the background issues remaining.  My example screenshots may have been unclear, please let me know if you want me to annotate those better or something.

File: common/components/controllers/CorporateHackathonController.jsx [hunk 3]
```diff
@@ -399,7 +402,6 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
             </div>
           </div>
         </div>
-        {this._renderContactSponsor()}
 
         <div className="corporate-sponsorship-saying corporate-bg-light col-12">
```
Comment (PeterBreen) [inline common/components/controllers/CorporateHackathonController.jsx:406]: This is an instance of `corporate-bg-light` to remove for the section background issue.

**Comment by schaferyan** [comment]
Hey Peter it looks like my latest changes weren't pushed to the feature branch. I had removed the TODO comment and resolved the background shading. I'll fix this as soon as I have a moment, apologies for that mistake.

File: common/components/controllers/CorporateHackathonController.jsx (hunk 2 above)
Comment (PeterBreen) [reply common/components/controllers/CorporateHackathonController.jsx:270]: I created issue #895 for this one. If there's any specifics you want to make sure we include in the documentation, please comment on it there.

File: common/components/controllers/CorporateHackathonController.jsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// democracylab.org/companies page --- TODO: rename controller
+:ww// democracylab.org/companies page --- TODO: rename controller
```
Comment (PeterBreen) [inline common/components/controllers/CorporateHackathonController.jsx:1]:
```suggestion
// democracylab.org/companies page --- TODO: rename controller
```
tiny clean-up item; removing this errant comment clears a build error.

**Review by PeterBreen — CHANGES_REQUESTED** [review]
One tiny thing left (see suggested change) and it looks great on both mobile and desktop in my testing. 👍

**Review by PeterBreen — APPROVED** [review]

**Pull request merged**